innkeeper/eskiptojson_test.go

```diff
@@ -0,0 +1,188 @@
+package innkeeper
+
+import (
+	"github.com/coreos/etcd/Godeps/_workspace/src/github.com/stretchr/testify/assert"
```

Comment:
fixed the import.

for this particular case, the asserts make the code more readable.

putting there a dozen of ifs would really make the code ugly.

table testing doesn't fit, as we check for different stuff in each of the tests...